python/lsst/sdm/schemas/efd_latiss.yaml

```diff
@@ -0,0 +1,2764 @@
+    "@id": "#exposure_efd.at_salindex205_wind_speed_maxmagnitude_min"
+    datatype: float
+    nullable: true
+    description: Minimum maximum wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex205_wind_speed_maxmagnitude_max"
+    "@id": "#exposure_efd.at_salindex205_wind_speed_maxmagnitude_max"
+    datatype: float
+    nullable: true
+    description: Maximum maximum wind speed magnitude in AuxTel-GillLabJack01 (salIndex 205).
+  - name: "at_salindex201_airflow_speed_mean"
+    "@id": "#exposure_efd.at_salindex201_airflow_speed_mean"
+    datatype: float
+    nullable: true
+    description: Mean median (mean for some sensors) wind speed in UNKNOWN (salIndex 201).
+  - name: "at_salindex201_airflow_speed_stddev"
+    "@id": "#exposure_efd.at_salindex201_airflow_speed_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the median (mean for some sensors) wind speed in UNKNOWN (salIndex 201).
+  - name: "at_salindex201_airflow_speed_min"
+    "@id": "#exposure_efd.at_salindex201_airflow_speed_min"
+    datatype: float
+    nullable: true
+    description: Minimum median (mean for some sensors) wind speed in UNKNOWN (salIndex 201).
+  - name: "at_salindex201_airflow_speed_max"
+    "@id": "#exposure_efd.at_salindex201_airflow_speed_max"
+    datatype: float
+    nullable: true
+    description: Maximum median (mean for some sensors) wind speed in UNKNOWN (salIndex 201).
+  - name: "at_salindex201_airflow_speedstddev_mean"
+    "@id": "#exposure_efd.at_salindex201_airflow_speedstddev_mean"
+    datatype: float
+    nullable: true
+    description: Mean standard deviation of wind speed estimated from quartiles in UNKNOWN (salIndex 201). Not available for all sensor types.
+  - name: "at_salindex201_airflow_speedstddev_stddev"
+    "@id": "#exposure_efd.at_salindex201_airflow_speedstddev_stddev"
+    datatype: float
+    nullable: true
+    description: Standard deviation of the standard deviation of wind speed estimated from quartiles in UNKNOWN (salIndex 201). Not available for all sensor types.
+  - name: "at_salindex201_airflow_speedstddev_min"
+    "@id": "#exposure_efd.at_salindex201_airflow_speedstddev_min"
+    datatype: float
+    nullable: true
+    description: Minimum standard deviation of wind speed estimated from quartiles in UNKNOWN (salIndex 201). Not available for all sensor types.
+  - name: "at_salindex201_airflow_speedstddev_max"
+    "@id": "#exposure_efd.at_salindex201_airflow_speedstddev_max"
+    datatype: float
+    nullable: true
+    description: Maximum standard deviation of wind speed estimated from quartiles in UNKNOWN (salIndex 201). Not available for all sensor types.
+  - name: "at_salindex201_airflow_direction_mean"
+    "@id": "#exposure_efd.at_salindex201_airflow_direction_mean"
+    datatype: float
+    nullable: true
+    description: Mean median (mean for some sensors) wind speed in UNKNOWN (salIndex 201).
```

Comment:
What is `UNKNOWN` here? Is this value missing in your YAML config? (Same for following columns with this value.)

Reply:
Fixed.